examples/discovery-search-app/cypress/integration/passages/spec.ts

```diff
@@ -58,7 +58,7 @@ describe('Passage Results', () => {
         cy.get('.bx--document-preview').should('exist');
         cy.get('.bx--document-preview')
           .contains(
-            'This is a document.\nThis result multiple passages, but you should only be able to see the first one. IBM if you can see this passage, something probably borked'
+            'This is a document. This result multiple passages, but you should only be able to see the first one. IBM if you can see this passage, something probably borked'
```

Comment:
This is necessary for this test to pass now since the Cypress version bump means that now: 

> `cy.contains()` ignores invisible whitespaces

per the [Migration Guide](https://docs.cypress.io/guides/references/migration-guide.html#cy-contains-ignores-invisible-whitespaces)